src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx

```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
 9. Optional: Set [**Violation time limit** for violations](#violation-time-limit) (this defaults to 24 hours).
 10. Select **Create**.
 
-## Create alert conditions for infrastructure integrations [#integration]
-
-To create an alert condition for your infrastructure integrations, use any of these options:
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "250px" }}>
-        UI options
-      </th>
-
-      <th>
-        Alert conditions for infrastructure integrations
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        From **Alerts & AI**
-      </td>
-
-      <td>
-        Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
+<Callout variant="important">
+  If New Relic hasn't received a Cloud Integration service's attribute in the past 60 minutes, then it won't be selectable for alerting in the UI. You can use the [API](/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/rest-api-calls-new-relic-infrastructure-alerts/) to create an alert condition for silent attributes.  
+</Callout>
 
-        OR
+## Other infrastructure alert condition methods [#methods]
+
+You can also use these other methods to create an infrastructure alert condition:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="alerts-ui"
+    title="Use the Alerts UI"
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="infrastructure-ui"
+    title="Use the Infrastructure UI"
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Infrastructure**, then select any of these Infrastructure monitoring pages: [**Hosts**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-compute-page-measure-resource-usage), [**Processes**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-processes-page-inspect-process-performance), [**Network**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-network-page-measure-compare-capacity), or [**Storage**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-storage-page-evaluate-disk-usage-efficiency). Mouse over the chart you want to alert on, select the ellipses <Icon name="fe-more-horizontal"/> icon, and then select **Create alert**.
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="infrastructure-settings"
```

Comment:
I wonder if you should call this id "integration" rather than "infrastructure-settings" because the original "integration" ID (line 57) has been deleted?